pages.it/linux/adduser.md

```diff
@@ -1,6 +1,7 @@
 # adduser
 
 > Servizio per aggiungere utenti.
+> Maggiori informazioni: <https://manpages.debian.org/latest/adduser/adduser.8.en.html>.
```

Comment:
Why not use <https://manned.org/adduser> here? Is it Debian specific? (Same for all other pages, except if the page is Debian specific.)

Reply:
Yes in my mind it was Debian specific, that's why I used debian manpages. Any one can confirm this please?

Reply:
I believe it is; it's also not in the Arch Linux man page DB.

Reply:
Then we should at least remove the locales from the links:
```suggestion
> Maggiori informazioni: <https://manpages.debian.org/latest/adduser/adduser.8.html>.
```

Reply:
@patricedenis will you please remove `.en` from all of the links?

Reply:
yes, no problemo. I wasn't online yesterday. :smile: